tests/QAN/timerange_test.js

```diff
@@ -39,6 +39,27 @@ Scenario(
   },
 );
 
+Scenario(
+  'PMM-T432 Open the QAN Dashboard and check that changing absolute time range updates the overview table, URL. @nightly @qan',
+  async ({ I, adminPage, qanDetails, qanFilters, qanOverview }) => {
+    const today = new Date();
+    const date = `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
```

Comment:
we have moment lib added at some point. Do you consider using moment for given strings since we already have it?

Reply:
thanks. i will use it